src/modules/ModuleLiquidity/View/PairsListItem.vue

```diff
@@ -16,6 +16,13 @@ const emit = defineEmits(['click:add', 'click:remove', 'click:deposit'])
 const { liquidityTokenBalance, pair } = toRefs(toRef(props, 'data'))
 const pairReactive = toReactive(pair)
 
+const balanceUsd = computed(() => {
+  const balance = liquidityTokenBalance.value
+  const { reserveUSD, reserveKLAY } = pairReactive
+  const price = new BigNumber(reserveUSD).dividedBy(reserveKLAY)
+  return price.isNaN() ? null : price.multipliedBy(balance)
```

Comment:
for lp token price is calculated as follows:
https://docs.starter.xyz/faq/how-to-guides/how-to-work-out-the-value-of-1-lp-token
https://dailydefi.org/articles/lp-token-value-calculation/
meaning: 
pair.reserveUSD / pair.totalSupply - price of 1 lp token
(pair.reserveUSD / pair.totalSupply) * liquidityTokenBalance.value - balance in usd

Reply:
Thank you very much!